aospy/region.py

```diff
@@ -42,20 +44,35 @@ def mask_var(self, data):
 
     @staticmethod
     def _get_land_mask(data, do_land_mask):
+        if not do_land_mask:
+            return 1
+        try:
+            land_mask = data.land_mask.copy()
+        except AttributeError:
+            # TODO: Implement aospy built-in land mask to default to.
```

Comment:
How do you envision this being done?  Would we take a reference land mask and just regrid it to whatever resolution was needed by the particular run (based on its lat and lon coordinates)?